<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="windowModality">
   <enum>Qt::ApplicationModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>779</width>
    <height>462</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="cursor">
   <cursorShape>ArrowCursor</cursorShape>
  </property>
  <property name="windowTitle">
   <string>OpenCV MewTracking</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QFrame" name="frameHSV">
    <property name="geometry">
     <rect>
      <x>9</x>
      <y>40</y>
      <width>511</width>
      <height>91</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::Box</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Sunken</enum>
    </property>
    <widget class="QWidget" name="verticalLayoutWidget">
     <property name="geometry">
      <rect>
       <x>140</x>
       <y>0</y>
       <width>131</width>
       <height>91</height>
      </rect>
     </property>
     <layout class="QVBoxLayout" name="layoutSliderMin">
      <item>
       <widget class="QSlider" name="sliderHUEmin">
        <property name="maximum">
         <number>179</number>
        </property>
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QSlider" name="sliderSATmin">
        <property name="maximum">
         <number>255</number>
        </property>
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QSlider" name="sliderVALmin">
        <property name="maximum">
         <number>255</number>
        </property>
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
    <widget class="QWidget" name="verticalLayoutWidget_2">
     <property name="geometry">
      <rect>
       <x>370</x>
       <y>0</y>
       <width>131</width>
       <height>91</height>
      </rect>
     </property>
     <layout class="QVBoxLayout" name="LayoutSliderMax">
      <item>
       <widget class="QSlider" name="sliderHUEmax">
        <property name="maximum">
         <number>179</number>
        </property>
        <property name="singleStep">
         <number>1</number>
        </property>
        <property name="value">
         <number>179</number>
        </property>
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QSlider" name="sliderSATmax">
        <property name="maximum">
         <number>255</number>
        </property>
        <property name="value">
         <number>255</number>
        </property>
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QSlider" name="sliderVALmax">
        <property name="maximum">
         <number>255</number>
        </property>
        <property name="value">
         <number>255</number>
        </property>
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
    <widget class="QWidget" name="verticalLayoutWidget_3">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>0</y>
       <width>121</width>
       <height>91</height>
      </rect>
     </property>
     <layout class="QVBoxLayout" name="layoutLBLmin">
      <item>
       <widget class="QLabel" name="lblHUEmin">
        <property name="text">
         <string>HUE &gt;&gt; MIN: 0</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="lblSATmin">
        <property name="text">
         <string>SAT &gt;&gt; MIN: 0</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="lblVALmin">
        <property name="text">
         <string>VAL &gt;&gt; MIN: 0</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
    <widget class="QWidget" name="verticalLayoutWidget_4">
     <property name="geometry">
      <rect>
       <x>300</x>
       <y>0</y>
       <width>61</width>
       <height>91</height>
      </rect>
     </property>
     <layout class="QVBoxLayout" name="layoutLBLmax">
      <item>
       <widget class="QLabel" name="lblHUEmax">
        <property name="text">
         <string>MAX: 179</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="lblSATmax">
        <property name="text">
         <string>MAX: 255</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="lblVALmax">
        <property name="text">
         <string>MAX: 255</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </widget>
   <widget class="QLabel" name="lblHSV">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>10</y>
      <width>111</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>16</pointsize>
     </font>
    </property>
    <property name="text">
     <string>HSV Filter</string>
    </property>
    <property name="textFormat">
     <enum>Qt::PlainText</enum>
    </property>
   </widget>
   <widget class="QLabel" name="lblRGB">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>160</y>
      <width>131</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>16</pointsize>
     </font>
    </property>
    <property name="text">
     <string>RGB Selector</string>
    </property>
    <property name="textFormat">
     <enum>Qt::PlainText</enum>
    </property>
   </widget>
   <widget class="QPushButton" name="btnCOLOURpicker">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>210</y>
      <width>81</width>
      <height>61</height>
     </rect>
    </property>
    <property name="text">
     <string>Select Colour</string>
    </property>
   </widget>
   <widget class="QFrame" name="frameRGBselector">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>200</y>
      <width>120</width>
      <height>80</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
   </widget>
   <widget class="QPushButton" name="btnAddColour">
    <property name="geometry">
     <rect>
      <x>530</x>
      <y>330</y>
      <width>61</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>ADD</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnRmColour">
    <property name="geometry">
     <rect>
      <x>690</x>
      <y>330</y>
      <width>61</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>REMOVE</string>
    </property>
   </widget>
   <widget class="QListWidget" name="listCOLOUR">
    <property name="geometry">
     <rect>
      <x>530</x>
      <y>40</y>
      <width>221</width>
      <height>281</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="lblRGBname">
    <property name="geometry">
     <rect>
      <x>310</x>
      <y>160</y>
      <width>111</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>16</pointsize>
     </font>
    </property>
    <property name="text">
     <string>RGB Name</string>
    </property>
    <property name="textFormat">
     <enum>Qt::PlainText</enum>
    </property>
   </widget>
   <widget class="QLineEdit" name="txtNAMEcolour">
    <property name="geometry">
     <rect>
      <x>310</x>
      <y>220</y>
      <width>131</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>14</pointsize>
     </font>
    </property>
   </widget>
   <widget class="QPushButton" name="btnEditColour">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>610</x>
      <y>330</y>
      <width>61</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>EDIT</string>
    </property>
   </widget>
   <widget class="QLabel" name="lblCOLOURlist">
    <property name="geometry">
     <rect>
      <x>530</x>
      <y>10</y>
      <width>111</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>16</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Colour List</string>
    </property>
    <property name="textFormat">
     <enum>Qt::PlainText</enum>
    </property>
   </widget>
  </widget>
  <widget class="QToolBar" name="toolBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionFIND_CAMERAS"/>
   <addaction name="actionCAM"/>
   <addaction name="actionHSV"/>
  </widget>
  <action name="actionCAM">
   <property name="text">
    <string>CAM OFF</string>
   </property>
   <property name="toolTip">
    <string>CAM OFF</string>
   </property>
  </action>
  <action name="actionHSV">
   <property name="text">
    <string>HSV OFF</string>
   </property>
   <property name="toolTip">
    <string>HSV OFF</string>
   </property>
  </action>
  <action name="actionFIND_CAMERAS">
   <property name="text">
    <string>FIND CAMERAS</string>
   </property>
   <property name="toolTip">
    <string>FIND CAMERAS</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
